spec/hbw/features/select_table_spec.rb

```diff
@@ -59,9 +71,48 @@
       expect_widget_presence
     end
 
-    scenario 'empty nullable multi = true' do
+    scenario 'empty bp variable, nullable, multi = true' do
       click_and_wait 'ORD-19'
 
+      real_initial_table_values = {
+        table_options:    table_options,
+        selected_options: selected_options
+      }
+
+      expected_initial_table_values = {
+        table_options:    ['Some name', 'Other name'].sort,
+        selected_options: []
+      }
+
+      expect(real_initial_table_values).to eq(expected_initial_table_values)
+      expect(page).to have_selector "button[type='submit']"
+
+      click_and_wait 'Submit'
+
+      expect(page).not_to have_content 'Field does not match regex'
+      expect(page).not_to have_content 'Field is required'
+
+      expect_widget_presence
+    end
+
+    scenario 'not empty bp variable, nullale, multi = true' do
+      click_and_wait 'ORD-41'
+
+      real_initial_table_values = {
+        table_options:          options_in_table_with_label('homsOrderDataSelect'),
+        selected_options:       selected_options_in_table_with_label('homsOrderDataSelect'),
+        table_options_multi:    options_in_table_with_label('homsOrderDataSelectMulti'),
+        selected_options_multi: selected_options_in_table_with_label('homsOrderDataSelectMulti')
+      }
```

Comment:
Why do you have two tables here?

Reply:
I've separated checking of two tables to different tests